### PR TITLE
add wang ergo to qmk

### PR DESCRIPTION
**Pull request by oceeean**
## Description

add wang ergo keyboard to qmk
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/ocean/wang_ergo/readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Wang Ergo
+
+![Wang Ergo](https://i.imgur.com/vsYWYfx.jpeg)
```
Comment (fauxpark) [inline keyboards/ocean/wang_ergo/readme.md:3]:
```suggestion
![Wang Ergo](https://i.imgur.com/vsYWYfxh.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by oceeean** [comment]
could you check the changes you requested? @fauxpark

**Review by fauxpark — APPROVED** [review]

**Comment by oceeean** [comment]
thank you 😁

**Comment by oceeean** [comment]
i misclicked :(

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**